File: Estrutura_Repetição/Python/Ex1071.py
x = int(input())
y = int(input())

menor = min(x, y)
maior = max(x, y)

if menor % 2 == 0:
    menor += 1
elif menor % 2 != 0:
    menor += 2

soma = 0
for i in range(menor, maior, 2):
    soma += i
print(soma)

File: Estrutura_Repetição/JavaScript/Ex1071.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift();};

var X = lines.shift()
var Y = lines.shift()

var maior = Math.max(X,Y)
var menor = Math.min(X,Y)
var soma = 0

for(let i = menor + 1; i < maior; i++){
    if(i%2 != 0){
        soma += i
    }
}
console.log(soma)